app/models/agents/webhook_agent.rb

```diff
@@ -69,5 +71,9 @@ def validate_options
     def payload_for(params)
       Utils.value_at(params, interpolated['payload_path']) || {}
     end
+
+    def response_message
+      options['response'] || 'Event Created'
```

Comment:
@cantino Yes. I'd really like to have the ability to allow an empty string.

I corrected the `interpolated` call to allow liquid templating